### PR TITLE
fix:  fix demo days button

### LINKED ISSUE #386: Bug: Text breaks for certain devices
### Describe the bug.

For certain device widths, the word 'contributors' breaks with s coming in next line.

### Describe the steps to reproduce the behavior.

Using inspect tools, check the webpage for Samsung Galaxy S8+.

### Expected behavior.

The word contributors should stay as one.

### Screenshots.

image

### Operating Environment

Windows 11. I tried reproducing the bug on Chrome.

### Additional Information

I would love to work on this issue if it is valid.

### PR DESCRIPTION
**Pull request by ktsrivastava29**
## Description

Fixes #386 #373 #405 

- [X] Bug fix (non-breaking change which fixes an issue).

## How Has This Been Tested?

Testing has been performed on Pop!_OS a linux based operation system
## 
Screenshot from 2023-01-18 19-50-15
Screenshot from 2023-01-18 19-51-14

...

## Checklist:

- [X] My code follows the style guidelines of this project.
- [X] I have performed a self-review of my own code.

### CODE REVIEW

**Review by padmajabhol — APPROVED** [review]

**Comment by padmajabhol** [comment]
@ktsrivastava29 please sign the DCO

**Comment by ktsrivastava29** [comment]
@padmajabhol ma'am I'm not able to find the DCO, could you kindly share it

**Comment by padmajabhol** [comment]
> @padmajabhol ma'am I'm not able to find the DCO, could you kindly share it

Check this out: https://github.com/moja-global/community-website/pull/391/checks?check_run_id=10727422831

**Comment by ktsrivastava29** [comment]
@padmajabhol ma'am I've committed and pushed the new changes with -s flag included, still I'm facing the dco error. How can I fix this. Should I close this PR and reopen a new one?

File: website/src/css/custom.css [hunk 1]
```diff
@@ -160,13 +160,23 @@ html[data-theme='dark'] .sponsor_wrapper:hover {
   background-color: #e3f2e8;
   border: none;
 }
+
```
Comment (sohamsshah) [inline website/src/css/custom.css:163]: please remove this extra line

File: website/src/css/custom.css [hunk 2]
```diff
@@ -160,13 +160,23 @@ html[data-theme='dark'] .sponsor_wrapper:hover {
...
 /* Custom Light */
 
 html[data-theme='light']
   .button.button--secondary.button--outline:not(.button--active):not(:hover) {
   color: var(--ifm-color-primary-white);
 }
 
+html[data-theme='light'] .demo {
+  color: #07903e;
```
Comment (sohamsshah) [inline website/src/css/custom.css:172]: is this color hex not available in our css vars? If not, then prefer the color that is already existing in our css var (checkout the `custom.css` file)

File: website/src/pages/index.js [hunk 3]
```diff
@@ -36,7 +36,11 @@ const HomepageHeader = () => {
             Watch the tutorials
           </Link>
           <Link
-            className={clsx('button button--outline button--primary button--lg', styles.getStarted)}
+            className={clsx(
+              'button button--outline button--primary button--lg demo',
```
Comment (sohamsshah) [inline website/src/pages/index.js:40]: where is this class `demo` applied?

**Review by sohamsshah — CHANGES_REQUESTED** [review]
Provided few minor comments.

File: website/src/css/custom.css (hunk 2 above)
Comment (ktsrivastava29) [reply website/src/css/custom.css]: done

File: website/src/pages/index.js (hunk 3 above)
Comment (ktsrivastava29) [reply website/src/pages/index.js:40]: in the demo button, in the landing page itself.

**Comment by ktsrivastava29** [comment]
@sohamsshah kindly look into it. So that I can move to other issues as well

**Pull request closed without merging**